### PR TITLE
Changes in HMI_API.xml

### PR DESCRIPTION
**Pull request by dtrunov**
Within investigation this defect I found problem in HMI_API.xml:
1.Structures for navigation request was in interface navigation I moved them to interface common,
2.minlength for image value was changed to 0 after discussion with Britanova A. and according with MOBILE_API.xml.
Defect isn't actual because lua script checks image value for minlength=1. I wrote comment in task APPLINK-24436 to change lua script.

Related issue [APPLINK-25255](https://adc.luxoft.com/jira/browse/APPLINK-25255)

### CODE REVIEW

**Comment by dtrunov** [comment]
Please review @AGritsevich @AGaliuzov @OHerasym @dev-gh @tmelnyk @TMelnyk @wolfylambova22

File: src/components/interfaces/HMI_API.xml [hunk 1]
```diff
@@ -3405,42 +3403,42 @@
       <description>If "true" - video data started. If "false" - video data stopped.</description>
     </param>
   </function>
-  <function name="GetWayPoints" functionID="GetWayPointsID" messagetype="request">
+  <function name="GetWayPoints" messagetype="request">
```
Comment (dev-gh) [inline src/components/interfaces/HMI_API.xml]: @dtrunov 
Why functionID is not used anymore?

**Comment by dev-gh** [comment]
Reviewed

File: src/components/interfaces/HMI_API.xml (hunk 1 above)
Comment (TMelnyk) [inline src/components/interfaces/HMI_API.xml]: functionID used **only** at MOBILE_API

Comment (dtrunov) [inline src/components/interfaces/HMI_API.xml]: @dev-gh functionID isn't used in HM_API.xml. They were added mistakenly.

**Comment by TMelnyk** [comment]
reviewed

**Comment by AGritsevich** [comment]
Reviewed.

**Comment by wolfylambova22** [comment]
Reviewed.

**Comment by AGaliuzov** [comment]
Reviewed

**Comment by OHerasym** [comment]
Reviewed

**Comment by AGritsevich** [comment]
@dtrunov, done.

**Pull request merged**